surface/src/on_nurbs/on_nurbs.cmake

```diff
@@ -1,4 +1,24 @@
 
+message(STATUS "EIGEN_VERSION: ${EIGEN_VERSION}")
+message(STATUS "EIGEN_WORLD_VERSION: ${EIGEN_WORLD_VERSION}")
+message(STATUS "EIGEN_MAJOR_VERSION: ${EIGEN_MAJOR_VERSION}")
+message(STATUS "EIGEN_MINOR_VERSION: ${EIGEN_MINOR_VERSION}")
+
+#set(PREPROCESSOR_EIGEN_VERSION "-DEIGENVERSION_GEQUAL_3_2")
```

Comment:
the implementation using Eigen::SparseSolver is not stable. This requires more sophisticated redesign of the package (it was created long before Eigen 3.2). Until then, this is commented out. if you'd like I can also remove it in my next commit